### ISSUE TITLE
Hello,

### ISSUE THREAD

**Issue by PaulinJak** [jorgenschaefer/elpy#1756]
# Summary

I want to use elpy in Emacs 26. 
Before Elpy was not finding the black, jedy, etc modules (although I installed them externally with pip). I resintalled emacs with snap, and now I cannot even open elpy-config...

# Steps to reproduce

# My configuration
## OS
Ubuntu 18.04.04 LTS
Emacs 26.3

## Result of `(elpy-config)`
I cannot open elpy-config, I get this error from 'virtual-env buffer':
'''
Running 'python -m venv /home/pjacquot/.emacs.d/elpy/rpc-venv':

Error: Command '['/home/pjacquot/.emacs.d/elpy/rpc-venv/bin/python', '-Im', 'ensurepip', '--upgrade', '--default-pip']' returned non-zero exit status 1.

Elpy failed to install its dedicated virtualenv due to the above
error. If the error details does not help you fixing it, You can
report this problem on Elpy repository on github.
In the meantime, setting the `elpy-rpc-virtualenv-path' option to
either `global' or `current' should temporarily fix the issue.
'''

## Elpy configuration in my init.el
```
;; MELPA Package Support

;; ===================================

;; Enables basic packaging support

(require 'package)


;; Adds the Melpa archive to the list of available repositories

(add-to-list 'package-archives

             '("melpa" . "http://melpa.org/packages/") t)


;; Initializes the package infrastructure

(package-initialize)

;; If there are no archived package contents, refresh them

(when (not package-archive-contents)

  (package-refresh-contents))

;; Installs packages

;;

;; myPackages contains a list of package names

(defvar myPackages

  '(better-defaults                 ;; Set up some better Emacs defaults
    elpy                            ;; Emacs Lisp Python Environment
    flycheck 
    material-theme                  ;; Theme

    )

  )

;; Scans the list in myPackages

;; If the package listed is not already installed, install it

(mapc #'(lambda (package)

          (unless (package-installed-p package)

            (package-install package)))

      myPackages)



(custom-set-variables
 ;; custom-set-variables was added by Custom.
 ;; If you edit it by hand, you could mess it up, so be careful.
 ;; Your init file should contain only one such instance.
 ;; If there is more than one, they won't work right.
 '(package-selected-packages (quote (material-theme better-defaults))))
(custom-set-faces
 ;; custom-set-faces was added by Custom.
 ;; If you edit it by hand, you could mess it up, so be careful.
 ;; Your init file should contain only one such instance.
 ;; If there is more than one, they won't work right.
 )
g
;; ====================================
;; Development Setup
;; ====================================

;; Enable elpy
(elpy-enable)


```

**Comment by galaunay** [comment]
Hi, 

Elpy is apparently failing to create its dedicated virtualenv.
This virtualenv is used to run the python packages that provide completion, documentation and other useful things.

Apparently, it is failing during the virtualenv creation, when this command is run:
```bash
/home/pjacquot/.emacs.d/elpy/rpc-venv/bin/python -Im ensurepip --upgrade --default-pip
```

Could you try running it from the shell, and post the result here ? It should give us some insight on what's going wrong.

As a temporary workaround, you can add the following to your `init.el\:
``` lisp
(setq elpy-rpc-virtualenv-path 'global)
```

**Comment by galaunay** [comment]
Just read another issue (#1760) mentioning this problem.
I though ensure-pip was builtin, but it is apparently not the case.
It should be fixed by installing  `python-venv` from apt (or `python3-venv`).

Please let me know if it works.

**Comment by galaunay** [comment]
I just merged 581a40ad23747fd500da194e7476f798289bd83a, that should address this issue.
Please re-open this if there is still a problem.
